.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## Description
```

Comment:
nit: Remove `## Description` header. This will make automated PR description snippets (e.g. in Teams) more useful.